src/bpm/bpm_extractor.py

```diff
@@ -1,8 +1,8 @@
 import sys
 import os
 from multiprocessing import Pool
-from tabulate import tabulate
 
+from tabulate import tabulate
```

Comment:
move up? pep8

Reply:
Is the correct place. Tabulate is an external package installed by pip